README.md

```diff
@@ -72,32 +57,68 @@ Password: `pmcdev`
-1. Duplicate `VVV/www/wordpress-trunk/public_html/tests` to your provisioned site, e.g. `VVV/www/sportico-com/public_html/tests`
-1. Duplicate and rename `VVV/www/wordpress-trunk/public_html/wp-tests-config-sample.php` to your provisioned site, e.g. `VVV/www/sportico-com/public_html/wp-tests-config.php`
+1. Install phpunit
+    ```bash
+    $ vagrant ssh
+    $ sudo mkdir -p /usr/share/php/phpunit
+    $ cd /usr/share/php/phpunit
+    $ sudo composer require --dev phpunit/phpunit ^7 --update-with-all-dependencies
+    $ sudo ln -sf /usr/share/php/phpunit/vendor/bin/phpunit /usr/bin/phpunit 
+    ```
+1. Create a testing database
+    ```bash
+    $ vagrant ssh
+    $ mysql -u root --password=root -e "CREATE DATABASE IF NOT EXISTS pmctests"
+    $ mysql -u root --password=root -e "GRANT ALL PRIVILEGES ON pmctests.* TO wp@localhost IDENTIFIED BY 'wp';"
+    ```
+1. Get the WP Test Suite
+    1. Note, WP Develop is usually further ahead in development than our stack—using its master branch (i.e. by provisioning in `wordpress-trunk` into VVV via https://github.com/Varying-Vagrant-Vagrants/custom-site-template-develop) can lead to issues when running tests with our pmc-unit-test bootstrap.php) As such, we manually clone wp trunk and switch to a compatable branch.
+    ```bash
+    $ vagrant ssh
+    $ git clone git@github.com:WordPress/wordpress-develop.git
+    $ cd wordpress-develop
+    
+    # As of Sept 2021 pmc-unit-test is compatable with WP 5.8
+    $ git checkout 5.8
+    ```
+1. Copy the WP Test Suite per site
+    ```bash
+    $ vagrant ssh
+    $ cp -r ~/wordpress-develop/tests/ /srv/www/sportico-com/public_html/
+    ```
+1. Create `wp-tests-config.php` per site
+    ```bash
+    $ vagrant ssh
+    $ cp ~/wordpress-develop/wp-tests-config-sample.php /srv/www/sportico-com/public_html/wp-tests-config.php
+    ```
     1. change line 4 to `define( 'ABSPATH', dirname( __FILE__ ) . '/' );`
+    1. comment line 12 i.e. `//define( 'WP_DEFAULT_THEME', 'default' );`
     1. Configure `DB_*` named constants:
         ```
-        define( 'DB_NAME', 'wordpresstrunk' );
+        define( 'DB_NAME', 'pmctests' );
         define( 'DB_USER', 'wp' );
         define( 'DB_PASSWORD', 'wp' );
         define( 'DB_HOST', 'localhost' );
         ```
-1. SSH into Vagrant `vagrant ssh`.
-1. Tell PHPUnit where our test bootstraps are located. Note, this must be done each time you SSH into vagrant (See below PHPStorm docs to automate this). Note, change `sportico-com` to the site you're testing within.
+    1. Add `define( 'PMC_IS_VIP_GO_SITE', true );`
+1. Run tests 
+    1. Note, we must tell PHPUnit where our test bootstraps are located. Note, this must be done each time you SSH into vagrant (See below PHPStorm docs to automate this). Note, change `sportico-com` to the site you're testing within.
     ```
+    $ vagrant ssh
+    
```

Comment:
I had an issue with this setting conflicting with this:

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/33085a35760adede17c21354ebc34068c8816f67/provision/core/env/homedir/.bash_aliases#L14-L20

maybe worth noting like

```suggestion
    You can add these variables to your bash, but make sure they are at the bottom of ~/.bash_alias as not to conflict with vvv's settings.
```